examples/typescript/package.json

```diff
@@ -6,11 +6,14 @@
   "license": "MIT",
   "dependencies": {
     "@ethersproject/wallet": "^5.7.0",
-    "@vocdoni/sdk": "0.0.12",
+    "@vocdoni/sdk": "^0.0.12",
     "chalk": "4.1.2"
   },
   "scripts": {
-    "start": "ts-node src/index.ts"
+    "start": "ts-node src/index.ts",
+    "quadratic": "ts-node src/quadratic.ts",
+    "approval": "ts-node src/approval.ts",
+    "linearweighted": "ts-node src/linear-weighted.ts"
```

Comment:
I would change `linear weighted` to `ranked`, it's more understandable for an average user

Reply:
Refactored 92c10334baf8ed39bbd3ef4f1b285fee15e6b943